test/googleAnalyticsService.spec.js

```diff
@@ -0,0 +1,93 @@
+const expect = require('chai').use(require('chai-dom')).expect,
+    sinon = require('sinon'),
+    {
+        JSDOM
+    } = require('jsdom'),
+    GoogleAnalyticsService = require('../web/assets/js/GoogleAnalyticsService'),
+    GoogleAnalyticsItem = require('../web/assets/js/GoogleAnalyticsItem');
+
+describe('GoogleAnalyticsService', () => {
+    let gaService,
+        document,
+        jQuery,
+        eventClick;
+
+    const template = '/html/gaTemplate.html',
+        gaDataAttr = 'data-ga-action',
+        gaCategoryAttr = 'data-ga-category',
+        gaLabelAttr = 'data-ga-label';
+
+    beforeEach(() => {
+        gaService = new GoogleAnalyticsService();
+        eventClick = sinon.spy();
+
+        return JSDOM.fromFile(__dirname + template)
+            .then(dom => {
+                document = dom.window.document
+                jQuery = require('jquery')(dom.window);
```

Comment:
Do you really need to use jQuery for listening to clicks? Why not `document.addEventListener('click', handleClick, false);`?